src/containers/FiveDay.js

```diff
@@ -1,20 +1,19 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router';
+// import { Link } from 'react-router';
```

Comment:
We shouldn't have commented code in master. If this feature is complete and we're not using this line, then just go ahead and delete it.